airqo-platform/AirQo-frontend
# MOB-175-remove unused svgs and replace with Material icons

#### Summary of Changes (What does this PR do?)
- Remove unused svgs and replace with Material icons

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
- Run the app,
- Evaluate the changed icons.
- You can also just crosscheck through the ap to ensure nothing is broken.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
